spec/factories/bus_stops.rb

```diff
@@ -4,5 +4,27 @@
   factory :bus_stop do
     name 'stop'
     sequence :hastus_id
+    trait :pending do
+      %i[name bench curb_cut lighting mounting mounting_direction
+        schedule_holder shelter sidewalk_width trash mounting_clearance
+        sign_type shelter_condition shelter_pad_condition shelter_pad_material
+        shelter_type shared_sign_post garage_responsible need_work obstructions
+        stop_sticker route_stickers bike_rack real_time_information].each do |s|
+        send(s, 'not blank')
```

Comment:
We should definitely be providing valid values in the factory. These are not valid, correct?